src/server/google-cloud.md

```diff
@@ -58,3 +67,6 @@ If you _want_ to use App Engine, consider using the [`appengine` package][].
 [server examples]: https://github.com/dart-lang/samples/tree/master/server
 [Docker images]: https://hub.docker.com/r/google/dart
 [GKE overview]: https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview
+[Dart Functions Framework]: https://pub.dev/packages/functions_framework
+[CloudEvents]: https://cloudevents.io/
```

Comment:
Just a thought here, but it might be more helpful to link to *Google's* supported CloudEvents here: https://github.com/googleapis/google-cloudevents. It's a repo, but the repo does point to cloudevents.io ...

Reply:
Hrm...but we don't implement any of these types (yet).

Do we have a better "this is what Cloud Events is" page?

Reply:
Hmm, good point. I think punt and leave it the way you had it for now...